domain/src/main/java/co/nimblehq/compose/crypto/domain/usecase/GetCoinDetailUseCase.kt

```diff
@@ -0,0 +1,15 @@
+package co.nimblehq.compose.crypto.domain.usecase
+
+import co.nimblehq.compose.crypto.domain.model.CoinDetail
+import co.nimblehq.compose.crypto.domain.repository.CoinRepository
+import kotlinx.coroutines.flow.Flow
+import javax.inject.Inject
+
+class GetCoinDetailUseCase @Inject constructor(private val repository: CoinRepository) {
```

Comment:
@luongvo What about if we want to set the `localization` to true 🤔 Should we create `GetCoinDetailUseCaseInput`?

Reply:
@Wadeewee I don't have that use case in this project so I think only having coinId param is fine